habitat/robots/static_manipulator.py

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+@attr.s(auto_attribs=True, slots=True)
+class StaticManipulatorParams:
+    """Data to configure a static manipulator.
+    :property arm_joints: The joint ids of the arm joints.
+    :property gripper_joints: The habitat sim joint ids of any grippers.
+    :property arm_init_params: The starting joint angles of the arm. If None,
+        resets to 0.
+    :property gripper_init_params: The starting joint positions of the gripper. If None,
+        resets to 0.
+    :property ee_offset: The 3D offset from the end-effector link to the true
+        end-effector position.
+    :property ee_link: The Habitat Sim link ID of the end-effector.
+    :property ee_constraint: A (2, N) shaped array specifying the upper and
+        lower limits for each end-effector joint where N is the arm DOF.
+    :property cameras: The cameras and where they should go. The key is the
+        prefix to match in the sensor names. For example, a key of `"robot_head"`
+        will match sensors `"robot_head_rgb"` and `"robot_head_depth"`
+    :property gripper_closed_state: All gripper joints must achieve this
+        state for the gripper to be considered closed.
+    :property gripper_open_state: All gripper joints must achieve this
+        state for the gripper to be considered open.
+    :property gripper_state_eps: Error margin for detecting whether gripper is closed.
+    :property arm_mtr_pos_gain: The position gain of the arm motor.
+    :property arm_mtr_vel_gain: The velocity gain of the arm motor.
+    :property arm_mtr_max_impulse: The maximum impulse of the arm motor.
+    """
+
+    arm_joints: List[int]
+    gripper_joints: List[int]
+
+    arm_init_params: Optional[np.ndarray]
+    gripper_init_params: Optional[np.ndarray]
+
+    ee_offset: mn.Vector3
```

Comment:
Does this need to be a vector 3? Can it not implicitly convert from the np.ndarray?